backends/occa/ceed-occa-qfunction.cpp

```diff
@@ -85,7 +85,7 @@ namespace ceed {
 
       // CEED defines
       props["defines/CeedPragmaSIMD"] = "";
-      props["defines/CEED_Q_VLA"] = 1;
+      props["defines/CEED_Q_VLA"] = "OCCA_Q";
```

Comment:
This definition for the VLA macro support looks reasonable but does not in fact work. I am pondering, but for now I've modified the TAP script to indicate that the OCCA backend still does not support the VLA macro for all modes.

Reply:
OCCA appears to not yet support
```
(*u)[8] = (const CeedScalar(*)[8])in[1];
```
@dmed256, is this correct?